light_curves/code_src/ztf_functions.py

```diff
@@ -97,23 +94,26 @@ def file_name(filtercode, field, ccdid, qid, basedir=None):
         assert len(files) == 1, f"found {len(files)} files. expected 1."
         return files[0]
 
-    f = f"{basedir}/field{field:06}/ztf_{field:06}_{filtercode}_c{ccdid:02}_q{qid}_{DATARELEASE}.parquet"
-    return CATALOG_ROOT + f
+    name = f"{basedir}/field{field:06}/ztf_{field:06}_{filtercode}_c{ccdid:02}_q{qid}_{datarelease}.parquet"
+    return CATALOG_ROOT + name
 
 
-def locate_objects(coords_list, labels_list, radius):
+def locate_objects(source_table, radius, datarelease=DATARELEASE, chunksize=10000):
```

Comment:
re `chunksize`, the review discussion from the previous PR has to be addressed here:

https://github.com/fornax-navo/fornax-demo-notebooks/pull/157#discussion_r1344765469